http-default-accounts-fingerprints-nndefaccts.lua

```diff
@@ -5862,7 +5870,7 @@ table.insert(fingerprints, {
            and response.body
            and response.body:find("Polycom", 1, true)
            and response.body:find("submitLoginInfo", 1, true)
-           and response.body:lower():find("<title>polycom - configuration utility</title>", 1, true)
+           and response.body:lower():find("<title>poly%l- %- configuration utility</title>", 1)
```

Comment:
I have committed a slightly different change in 51b9aaa3d08698c63bcc693c46293131838666a2